src/main/java/uk/gov/hmcts/reform/managecase/PropertyLogger.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.managecase;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.core.env.AbstractEnvironment;
+import org.springframework.core.env.EnumerablePropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.MutablePropertySources;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.stream.StreamSupport;
+
+@Component
+public class PropertyLogger {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PropertyLogger.class);
+
+    @EventListener
+    public void handleContextRefresh(ContextRefreshedEvent event) {
```

Comment:
where do we use this?

Reply:
It was just a temporary one used for debugging some app config issues